include/firefly/memory-manager/mm.hpp

```diff
@@ -9,6 +9,12 @@ enum AddressLayout : uint64_t {
     Low = 0x0000000000000000UL
 };
 
+enum PageSize : uint64_t {
+	SIZE_4KB = 0x00001000,
+	SIZE_2MB = 0x00200000,
+	SIZE_1GB = 0x40000000,
+};
+
 static constexpr uint32_t PAGE_SIZE = 4096;
```

Comment:
We probably don't need this anymore.
The buddy and some other allocator components use it, but since they always operate on 4kib boundaries they can use `PageSize::Size4K`